lib/cache/cache.go

```diff
@@ -365,6 +365,16 @@ func ForDiscovery(cfg Config) Config {
 	return cfg
 }
 
+// ForPlugins sets up watch configuration for plugins servers.
+func ForPlugins(cfg Config) Config {
+	cfg.target = "plugins"
+	cfg.Watches = []types.WatchKind{
+		{Kind: types.KindAccessRequest},
```

Comment:
@mdwn I remember you were telling me that there was some issue with watching access request resources through the cache. Do you remember what it was?

Instead of watching access requests, I think we should instead use the events watcher like other plugins do, correct?

https://github.com/gravitational/teleport-plugins/blob/2ad68adbf5e3e09f2a493fed7de14aa722105409/access/pagerduty/app.go#L106-L112

https://github.com/gravitational/teleport/blob/master/integrations/lib/watcherjob/watcherjob.go

cc @fspmarshall

Reply:
We can watch access request resources, but we can't actually perform gets on access request resources. I'm basically adding in an access request watcher in auth to get around it.

Reply:
Okay, regardless @EdwardDowling I don't think we need the access requests cache in the service and can instead use the same watcher as other plugins? @justinas already moved it to integrations/lib in teleport repo (see link above).